apps/settings/js/panels/call/call_forwarding.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * Make sure all operations would be executed one by one.
+ *
+ * @module call/call_forwarding
+ */
+define(function(require) {
+  'use strict';
+
+  var Module = require('modules/base/module');
+  var EventEmitter = require('modules/base/event_emitter');
+  var CallUtils = require('panels/call/call_utils');
+  var CallConstant = require('panels/call/call_constant');
+
+  /**
+   * @class CallForwarding
+   * @returns {CallForwarding}
+   */
+  var CallForwarding = Module.create(function() {
+    this.super(EventEmitter).call(this, [
+      'set-callForwarding-invalid',
+      'set-callForwarding-updating',
+      'set-callForwarding-success',
+      'set-callForwarding-error',
+      'update-callForwarding-subpanels',
```

Comment:
These four events are considered as view related and should not be included in this module.